crates/prune/prune/src/segments/receipts.rs

```diff
@@ -1,5 +1,5 @@
 use crate::{
```

Comment:
this needs module docs here in place, to explain the fs hierarchy, i.e. when it can be pruned by user (still not sure about this word? by full node is what you mean? pls clarify in docs) and that it's pruned after static file production too